start_esp/start_esp.py

```diff
@@ -968,8 +968,6 @@ def enforce_conflict_args(args):
             return "Flag --enable_backend_routing cannot be used together with --service_control_url_override."
         if args.metadata != METADATA_ADDRESS:
             return "Flag --enable_backend_routing cannot be used together with --metadata."
-        if args.service_json_path:
```

Comment:
This flag wasn't added to the args parser?

Reply:
this line is for not allowing this flag if enable_backend_routing is true.  In ESP, enable_backend_routing=true will disable many flags that will not be supported by next version.
We should support service_json_path flag

Reply:
Yes, it was there.